Frontend/ui-driver/src/Screens/ReferralScreen/Handler.purs

```diff
@@ -29,11 +29,12 @@ import Types.ModifyScreenState (modifyScreenState)
 import Data.Maybe(isJust)
 import Screens.Types (ReferralType(..))
 import Screens.ReferralScreen.ScreenData as ReferralScreenData
+import Merchant.Utils(getMerchant, Merchant(..))
 
 referralScreen:: FlowBT String REFERRAL_SCREEN_OUTPUT
 referralScreen = do
   (GlobalState state) <- getState
-  action <- lift $ lift $ runScreen $ ReferralScreen.screen state.referralScreen{ props{ stage = LeaderBoard } }
```

Comment:
why we are not showing driver leaderboard ?

Reply:
Logic has been moved to flow.purs